inst/templates/derivationchain/body.mustache

```diff
@@ -1,8 +1,20 @@
 <div id="cy"></div>
 <br/>
 <p><i>Time-Series processor icon images courtesy of Aquatic Informatics.</i></p>
+<div id="legendContainer"><strong>Legend</strong><br/><br/>
+<span style="text-decoration:underline">Processor Types:</span> <br/>
+<img src="data:image/jpeg;base64,{{derivationChainSymbols.ratingModelDerived}}" title="Rating derived" alt="Rating derived"> Rating derived<br/>
```

Comment:
Once you move the functions to derivationchain-data.R, you should be able to access the same information with reportData.derivationChainSymbols.ratingModelDerived